qemu/tests/cpu_topology_test.py

```diff
@@ -46,7 +46,7 @@ def check(p_name, exp, check_cmd):
     for vcpu_threads in vcpu_threads_list:
         vcpu_sockets = min(max(host_cpu // (vcpu_cores * vcpu_threads), 1),
                            random.randint(1, 6))
-        vcpu_sockets = 2 if (os_type == 'Windows' and
+        vcpu_sockets = 2 if (os_type.lower() == 'windows' and
```

Comment:
`os_type` should always be `windows`.
https://github.com/avocado-framework/avocado-vt/blob/c4c4529d5c4f3d095f42f28a3cb4f9f4856b97cb/virttest/shared/cfg/guest-os/Windows.cfg#L6

Reply:
> `os_type` should always be `windows`. https://github.com/avocado-framework/avocado-vt/blob/c4c4529d5c4f3d095f42f28a3cb4f9f4856b97cb/virttest/shared/cfg/guest-os/Windows.cfg#L6

@PaulYuuu  ok, thank you, has updated the commit message, please help to review this, thanks.